cernopendata/modules/fixtures/cli.py

```diff
@@ -80,27 +85,27 @@ def _handle_record_files(record, data):
         if "type" in file and file["type"] == "index.json":
             print(
                 datetime.now(),
-                "This is an index file. Let's check the entries that it has:",
-                file.get("uri"),
+                "    This is an index file. Let's check the entries that it has.",
+                end=" ",
             )
             # We don't need to store the index
             FileIndexMetadata.create(
                 record, f, description=file.get("description", filename)
             )
-            print("File index created")
             f.delete()
         elif "type" in file and file["type"] == "index.txt":
             # The txt indexes should be ignored. Just delete the file
             f.delete()
         else:
             real_files.append(file)
             try:
-                obj = ObjectVersion.create(record.bucket, filename, _file_id=f.id)
+                obj = FileObjectCold.create(record.bucket, filename, f.id)
```

Comment:
From https://github.com/cernopendata/cernopendata-portal/pull/138
```suggestion
                obj = FileObjectCold.create_version(record.bucket, filename, f.id)
```